### ISSUE TITLE
saving is slow since 1.4

### ISSUE THREAD

**Issue by nikopp** [codinguser/gnucash-android#202]
Since the update to 1.4 every save I do (add or change a transaction) takes about 10 seconds on my HTC Desire S. (1.4.3 did not fix it)

Before the update it took less than a second.

Please let me know if you need more information or if I can do some testing to help find the problem.

**Comment by nikopp** [comment]
Hi again,

in the meantime I got the Sony Xperia Z3 compact and the problem also appears on that phone although it's high end hardware.

Is there any progress on this issue?

Can I help somehow?

**Comment by codinguser** [comment]
Not much, it is hard to replicate the issue and that is what is stalling progress on fixing it. 
So far, my observation is that one either experiences it, or not. 
We will review the code to eliminate any potential bottlenecks and hopefully that helps.

**Comment by nikopp** [comment]
I exported all transactions and attached the qif file after replacing the
names by rather meaningless terms.

I hope that helps in finding the bug.

**Comment by codinguser** [comment]
@nikopp is this still an issue?

**Comment by nikopp** [comment]
Some days ago I did an export because I noticed that it became slower again.

I will observe the behaviour and comment again in a couple of weeks.

Furthermore I noticed, that the attachment from my comment on 10 Apr was not attached to the issue because I sent it via mail and the file type is not supported.

I did the upload again as a TXT but it is in fact a QIF.
[20150410_094837_gnucash_export_EUR.qif.txt](https://github.com/codinguser/gnucash-android/files/41161/20150410_094837_gnucash_export_EUR.qif.txt)
